continuous-integration/check.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash -e
+
+# Get and install dependencies
+echo
+echo '-- Prepare --'
+echo "Running on: $DISTRO $OS"
+cd /cwd
+
+if [[ $DISTRO == "ubuntu" ]]; then
+    apt-get update -qq
+    apt-get -y install build-essential
+    apt-get -y install libmount-dev libpcre3-dev python-dev dh-autoreconf flex bison pkg-config zlib1g-dev libffi-dev
+    apt-get -y install libxml2-utils libgamin-dev
+    apt-get -y install software-properties-common
+    add-apt-repository -y ppa:ricotz/testing
+    apt-get update
+    apt-get -y install libmozjs-31-dev
+else
+    exit 1
+fi
+
+# Build glib 2.0
```

Comment:
> No tricks are allowed; Or, whatever it does is acceptable?

What kind of tricks do you mean?

> All distros out there uses jhbuild when packaging GNOME?

Not when packaging, likely. But building the master branch of any GNOME module is probably going to require unreleased commits from any number of other GNOME modules. GJS is probably exceptionally easy that it only requires gobject-introspection and GLib (and the version of GLib is actually already in Ubuntu 16.10 and Fedora 25 I think.)